app/views/layouts/_segment_analytics.html.erb

```diff
@@ -0,0 +1,19 @@
+<!-- Global site tags for Segment analytics-->
+<script>
+  const SEGMENT_IDS = {
+    // TODO: Replace these with new accounts after they get set up
+    // These are designed to be public: https://community.segment.com/t/m26sng/writekey-accessible-by-anyone
+    prod: "7gUE8JMXEkhEo9ioD1uHke1RgpDFxxlG",
+    staging: "7gUE8JMXEkhEo9ioD1uHke1RgpDFxxlG",
+    development: "7gUE8JMXEkhEo9ioD1uHke1RgpDFxxlG"
+  };
+
+  // Snippet from https://segment.com/docs/sources/website/analytics.js/quickstart/
+  const matched_id = SEGMENT_IDS["<%= Rails.env %>"];
+  if (matched_id) {
+    !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
+      analytics.load(matched_id);
+      analytics.page();
```

Comment:
Did you review the optional properties here https://segment.com/docs/sources/website/analytics.js/#page and here https://segment.com/docs/sources/website/analytics.js/#load-options to see if any are relevant?

Reply:
And to what extent should we use analytics.page vs analytics.track to record something like a SAMPLE_REPORT_VIEW ? Does the SAMPLE_REPORT_VIEW page have a unique URL?

Reply:
- Yes, I don't find the optional fields that interesting yet but we could add some later.
- I would propose that our team relies on the "tracks" primarily because those will represent the most explicit actions (e.g. there could be a series like "sample_upload_selected", "sample_upload_started") but I think the "page" info could be useful as a secondary source.

Reply:
Sounds good 👍